src/app/prt-staking/components/prt-section/prt-widget.tsx

```diff
@@ -101,14 +124,15 @@ export function PrtWidget({ token, onClose }: Props) {
         formattedFiat=''
         selectedToken={selectedToken}
         selectedTokenAmount={inputAmount}
+        showSelectorButton={currentTab !== WidgetTab.CLAIM}
         showSelectorButtonChevron={false}
         onChangeInput={(_, amount) => setInputAmount(amount)}
         onClickBalance={onClickBalance}
         onSelectToken={() => {}}
       />
       <TradeButton
         label={buttonLabel}
-        isDisabled={false}
+        isDisabled={currentTab === WidgetTab.STAKE && !canStake}
```

Comment:
Leverage the new `canStake` functionality (ensure the user does not stake during the buffer time)